src/modules/organisation-role/organisation-role.controller.ts

```diff
@@ -89,6 +94,18 @@ export class OrganisationRoleController {
     }
   }
 
+  @Delete(':id/roles:roleId')
+  @UseGuards(AuthGuard, OwnershipGuard)
```

Comment:
AuthGuards are implemented by default
```suggestion
@UseGuards(OwnershipGuard)
```